### PR TITLE
api: Add users.me.getProfile() endpoint.

### LINKED ISSUE #32: users: Add getProfile method.
We should add a `getProfile` method to [users.js](https://github.com/zulip/zulip-js/blob/master/src/resources/users.js) that allows one to get the profile of the user/bot that requests the `GET /api/v1/users/me` endpoint, similar to [the `get_profile` method](https://github.com/zulip/python-zulip-api/blob/master/zulip/zulip/__init__.py#L735) in our Python API.

@aero31aero: FYI

Also, see, zulip/zulip#7994.

### PR DESCRIPTION
**Pull request by Balaji2198**
Fixes #32.

### CODE REVIEW

**Comment by Balaji2198** [comment]
@aero31aero  Thanks for the review. Updated.

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ zulip.callEndpoint('/messages', 'POST', params);
 | `zulip.typing.send()` | POST `/typing` | can be used to send a typing notification. The parameters required are `to` (either a username or a list of usernames) and `op` (either `start` or `stop`). |
 | `zulip.users.retrieve()` | GET `/users` | retrieves all users for this realm. |
 | `zulip.users.me.pointer.retrieve()` | GET `/users/me/pointer` | retrieves a pointer for a user. The pointer is the id of the last message the user read. This can then be used as an anchor message id for subsequent API calls. |
+| `zulip.users.me.getProfile.retrieve()` | GET `/users/me/getProfile` | retrieves the profile of the user/bot. |
```
Comment (aero31aero) [inline README.md]: Sorry for bugging you for minor changes. This one's left. :)

Comment (Balaji2198) [reply README.md]: @aero31aero  Really sorry for that! Hope everything is fine now.

**Comment by aero31aero** [comment]
Merged! Thanks @Balaji2198 !

**Pull request merged**